pipelines/build/openjdk_pipeline.groovy

```diff
@@ -190,14 +190,14 @@ node('worker') {
     }
 
     // Load buildConfigFilePath. This is where jdkxx_pipeline_config.groovy is located. It contains the build configurations for each platform, architecture and variant.
-    def buildConfigFilePath = (params.buildConfigFilePath) ?: "${DEFAULTS_JSON['configDirectories']['build']}/${javaToBuild}_pipeline_config.groovy"
+    def buildConfigFilePath = (params.buildConfigFilePath) ?: "${DEFAULTS_JSON['configDirectories']['build']}/${javaToBuild}u_pipeline_config.groovy"
```

Comment:
If the u_pipeline_config isn't there, what does buildConfigFilePath fall back to?

Reply:
If it isn't there, then the if statement in the following lines of code should notice that and default to the non-u pipeline config file.